analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ExecutingSqlQueriesBase.cs

```diff
@@ -52,7 +52,8 @@ public abstract class ExecutingSqlQueriesBase<TSyntaxKind, TExpressionSyntax, TI
                 KnownType.MySql_Data_MySqlClient_MySqlDataAdapter,
                 KnownType.MySql_Data_MySqlClient_MySqlScript,
                 KnownType.Microsoft_Data_Sqlite_SqliteCommand,
-                KnownType.Microsoft_EntityFrameworkCore_RawSqlString
+                KnownType.Microsoft_EntityFrameworkCore_RawSqlString,
+                KnownType.Dapper_CommandDefinition,
```

Comment:
The convention is not to use trailing commas, following the less is more. Also below